features/step_definitions/work_steps.rb

```diff
@@ -216,19 +216,30 @@
   Work.tire.index.refresh
 end
 
+When /^a chapter with the co-author "([^\"]*)" is added to "([^\"]*)"$/ do |coauthor, work_title|
```

Comment:
As noted elsewhere, the `([^\"]*)` could be `([^"]*)` or `(.*)` but not a big deal as we have this all over the place